ports.go

```diff
@@ -0,0 +1,74 @@
+package cagent
+
+import (
+	"fmt"
+	"syscall"
+
+	"github.com/shirou/gopsutil/net"
+	log "github.com/sirupsen/logrus"
+)
+
+type PortStat struct {
+	Protocol     string `json:"proto"`
+	LocalAddress string `json:"addr"`
+	PID          int32  `json:"pid,omitempty"`
+	ProgramName  string `json:"program,omitempty"`
+}
+
+func (ca *Cagent) PortsResult(processList []ProcStat) (m MeasurementsMap, err error) {
+	connections, err := net.Connections("inet")
+	if err != nil {
+		log.Error("[PORTS] error: ", err.Error())
```

Comment:
Logging an error is basically handling it. Return the error without logging here and let the caller of this function log the error

Reply:
caller is handler.GetAllMeasurements() 
This function does not do any logging at all. It's just checking if there were any errors.

I agree that the caller should do the logging, not the child, but I think it's better to implement it as separate PR for all calls from within GetAllMeasurements() function.